src/server.py

```diff
@@ -482,8 +486,8 @@ def run_seta_details_query():
                 datetime=time_of_now)
             conn.execute(statement)
 
-            # we must reset the delta after we update the datetime
-            delta = 0
```

Comment:
we don't need to update delta to 0 due to it will been updated every push. And we could use it to see if we meet the 90 minutes limit in https://github.com/mozilla/ouija/pull/199